README.md

```diff
@@ -34,9 +43,8 @@ Parameter | Type | Data Type | Description | Required
 | cors | query | string | CORS support for entry ("yes", "no", or "unknown") | No |
 | category | query | string | return entries of a specific category | No |
 
-## **GET** /health-check
+For example, https://api.publicapis.org/entries?title=Do
```

Comment:
I'd prefer if the example was a bit more verbose, perhaps `/entries?category=development&https=true`.